docs/book/src/reference/versions.md

```diff
@@ -132,7 +132,7 @@ The Kubeadm Control Plane requires the Kubeadm Bootstrap Provider.
 | v0.4 (v1alpha4) | v1.8.4                          |
 | v1.0 (v1beta1)  | v1.8.5                          |
 | v1.1 (v1beta1)  | v1.9.3                          |
-| v1.2 (v1beta1)  | v1.9.3 (v1.10.0 with >= v1.2.7) |
+| v1.2 (v1beta1)  | v1.9.3 (v1.10.1 with >= v1.2.7) |
```

Comment:
I think this is not correct. It should show the specific patch version with which we support v1.10.1

(similar in other PRs)

Reply:
Maybe we should add additional lines instead of trying to put the patch versions in the column

Reply:
:facepalm: You're right.

Not sure what the best way to represent this is, or if we should just have a single version for the latest patch and mention that limitation in a note?

Reply:
I think I like something like this:
```
| CAPI Version         | Max CoreDNS Version for Upgrade |
|----------------------|---------------------------------|
| v0.3 (v1alpha3)      | v1.8.4                          |
| v0.4 (v1alpha4)      | v1.8.4                          |
| v1.0 (v1beta1)       | v1.8.5                          |
| v1.1 (v1beta1)       | v1.9.3                          |
| v1.2 (v1beta1)       | v1.9.3 <br/>                    |
| >= v1.2.7 (v1beta1)  | v1.10.0                         |
| >= v1.2.11 (v1beta1) | v1.10.1                         |
| v1.3 (v1beta1)       | v1.10.0                         |
| v1.4 (v1beta1)       | v1.10.0                         |
```